holidayextras/holidayextras.github.com
# Optimise page speed

#### What does this PR do?
- Move all external stylesheet calls to the bottom of the page to stop rendering blocking on load
- Ensure all tap sizes are a minimum of 44px high
- Adds twitter cards

#### How should a developer review this?
- Checkout branch
- Run `jekyll serve` this should boot up the blog locally

#### How should this be manually tested?
- Hopefully you shouldn't notice too much of a change with page speed as it was fast already but when it is live you can visit <https://developers.google.com/speed/pagespeed/insights/> and check it rates highly according to google for page speed
- View page source and check the `<head> of a few pages and search for `twitter` each page should have its own twitter title and description
- On mobile everything should be tappable with a fat finger.
 
#### Screenshots (if appropriate)
![screen shot 2015-01-19 at 11 18 18](https://cloud.githubusercontent.com/assets/778942/5799715/81efe618-9fcd-11e4-8c91-23f667d8d856.png)

#### What gif best describes how you feel about this work?
![bigtrainfathandedtw t](https://cloud.githubusercontent.com/assets/778942/5799927/10d5a8f2-9fd0-11e4-9a8e-aada9903e338.gif)

---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer, Software Architect or Developer review:
